### ISSUE TITLE
Change autocomplete language dynamically

### ISSUE THREAD

**Issue by VerhoevenWout** [xkjyeah/vue-google-maps#281]
Hi there,
I'm working on a multi-language website and want to set the autocomplete language dynamically.
Is this possible?

Currently my code looks like this:
Vue.use(VueGoogleMaps, {
  load: {
    key: 'XXX',
    libraries: 'places',
    language: 'fr',
  }
});
Vue.component('gmap-autocomplete', VueGoogleMaps.Autocomplete);
Vue.component('google-map', VueGoogleMaps.Map);

Can I do something like this?
VueGoogleMaps.loaded.then(() => {
	LANGUAGE SETTING...
});

Thanks in advance!

**Comment by philmillman** [comment]
I don't think this is possible since you need to load a new js file for every country/region 

https://github.com/xkjyeah/vue-google-maps/blob/vue2/src/manager.js#L82

If I understand correctly you'll need to reload the plugin whenever the language or region changes so it can load the appropriate localized bundle from Google. 

More info here: https://developers.google.com/maps/documentation/javascript/localization

**Comment by danon** [comment]
So, why can't this feature be implemented?

**Comment by xkjyeah** [comment]
Because you can only load one JS file per page?

**Comment by danon** [comment]
And why is that? Conditional es6 imports work fine for me.

**Comment by xkjyeah** [comment]
Mmm, because we're not loading an ES6 module. We're loading Google Maps'
API.

**Comment by danon** [comment]
I asked why is that that we can only load one js file?

**Comment by xkjyeah** [comment]
If you try loading the API twice (once for each different language) Google
Maps will Bork. Don't ask me why -- ask Google

**Comment by danon** [comment]
What do you mean it"ll Bork? Return http error code and not return the file?

**Comment by xkjyeah** [comment]
Dude, why don't you try inserting two lines of Autocomplete into your HTML
and let me know? Do you JS?
